subgraphs/ens-governance/src/helpers.ts

```diff
@@ -0,0 +1,139 @@
+import { Address, BigDecimal, BigInt, log } from "@graphprotocol/graph-ts";
+import {
+  Delegate,
+  Governance,
+  Proposal,
+  TokenHolder,
+} from "../generated/schema";
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+export const BIGINT_ZERO = BigInt.fromI32(0);
+export const BIGINT_ONE = BigInt.fromI32(1);
+export const BIGINT_FIVE = BigInt.fromI32(5);
+export const BIGDECIMAL_ZERO = new BigDecimal(BIGINT_ZERO);
+
+export const GOVERNANCE_NAME = "OZGovernor";
+export const PROPOSAL_STATE_PENDING = "PENDING";
```

Comment:
I feel like something like this is cleaner:
``` typescript
export namespace ProposalState {
  export const PENDING = "PENDING";
  ...
}
```

Reply:
Agh this is much cleaner. I was using enums before but AssemblyScript wasn't very happy with me